code/processes/metrics.q

```diff
@@ -35,49 +41,39 @@ metrics:{[syms]
    / add allday window
    if[.metrics.enableallday;
     if[not all syms in key .metrics.start;.metrics.start::exec first time by sym from sumstab];
-    t:`sym`timediff xasc t,select sym,timediff:0Nn,vwap:sumsps%sumssize,twap:sumspricetimediff%.z.p - .metrics.start[sym] from latest where sym in syms];
-
-   t
-  
+    t:`sym`timediff xasc t,select sym,timediff:0Nn,vwap:sumsps%sumssize,twap:sumspricetimediff%.z.p - .metrics.start[sym] from latest where sym in syms];t  
```

Comment:
Hard to see that `t` is being returned here, I think for clarity this can be changed to:
```
if[.metrics.enableallday;
  if[not all syms...
  t:`sym`timediff...
];
:t;
```
I think using 2 space minimum will make this easier to read + having `:t;` on a newline makes it easier to see where the value is returned.